PolyEngine/Core/Src/Queue.hpp

```diff
@@ -79,7 +79,7 @@ namespace Poly
 		bool operator!=(const Queue<T>& rhs) const { return !(*this == rhs); };
 
 		//------------------------------------------------------------------------------
-		bool IsEmpty() const { return GetSize() == 0; }
+		bool Empty() const { return GetSize() == 0; }
```

Comment:
Why the rename? In Dynarray the method is called `IsEmpty()` let's revert this change and keep it consistent.